README.md

```diff
@@ -0,0 +1,15 @@
+# Homebrew formulas
+
+This repository contains automatically generated [homebrew]
+[formulas] for various Foxygoat tools and CLIs.
+
+[homebrew]: https://brew.sh
+[formulas]: https://docs.brew.sh/Formula-Cookbook
+
+The formulas can be used with 
+
+	brew tap foxygoat/tap
```

Comment:
These are indented with tab, not 4 spaces. I can't remember what, if anything, we decided about that, but I know I always use 4 spaces. The markdown guide says "indenting lines by four spaces or one tab" so either is fine.

Reply:
fixed.